src/test/java/org/primeframework/mvc/ManagedCookieTest.java

```diff
@@ -33,6 +34,43 @@
 public class ManagedCookieTest extends PrimeBaseTest {
   @Inject private Encryptor encryptor;
 
+  @Test
+  public void broken_csrf_case() throws Exception {
+    // When base64-decoded, the beginning of this cookie value is "5\n". When a cookie does not contain the header bytes, we attempt to
```

Comment:
good explanation. Is it useful to add a tiny bit more higher level stuff to explain how we got into this? That way, in the future, we know the exact use case this is covering at a higher level.